graphics/blender/Portfile

```diff
@@ -0,0 +1,208 @@
+categories          graphics multimedia
+platforms           darwin
+license             GPL-2+
+maintainers         @jasonliu--
+
+homepage            https://www.blender.org
+description         3-D computer graphics creation suite
+long_description    Blender is a ${description}. It contains a set of \
+                    tools that can be used for all phases of the 3-D \
+                    computer graphics pipeline, including modeling, \
+                    sculpting, texturing, rigging, animation, \
+                    simulation, rendering, compositing, motion \
+                    tracking, and video editing. It can be used for \
+                    creating animated films, visual effects, art, 3-D \
+                    printing, motion graphics, interactive 3-D \
+                    applications, and computer games.
+
+master_sites        https://download.blender.org/source/
+use_xz              yes
+checksums           rmd160  477b18c4d51ed6e5214158f1438c2c9a4317b2e7 \
+                    sha256  4de00030201d14e98cce3d3ed61b3c72267fb07dbd46ba779dc261a3d56beba3 \
+                    size    39890216
+
+compiler.cxx_standard 2011
+# We need to blacklist Xcode Clang/LLVM, because Blender needs to
+# use tools such as llvm-config, which are not included in Xcode.
+# (llvm-config is needed because Blender links to libLLVM. libLLVM is
+# also not included in Xcode.)
+# (Reference: https://stackoverflow.com/questions/32959627/where-is-the-system-llvm-config-on-macos)
+compiler.blacklist clang
+# Even though we need to blacklist Xcode Clang/LLVM, we still need to
+# make the rest of Xcode available to the build system. If we don't set
+# use_xcode to yes, then the build will fail, because Blender's CMake
+# scripts still need to use other parts of Xcode (e.g., the xcodebuild
+# command) besides just Xcode Clang.
+use_xcode yes
+cmake.install_prefix ${applications_dir}
+
+depends_lib-append  port:python38 \
+                    port:openal-soft \
+                    port:alembic \
+                    port:opensubdiv \
+                    port:jack \
+                    port:libsndfile \
+                    port:fftw-3 \
+                    port:zlib \
+                    port:freetype \
+                    port:openexr \
+                    port:ffmpeg \
+                    port:opencollada \
+                    port:pcre \
+                    port:libsdl2 \
+                    port:libpng \
+                    port:openjpeg \
+                    port:tiff \
+                    port:boost \
+                    port:openimageio \
+                    port:opencolorio \
+                    port:openvdb \
+                    port:osl \
+                    port:embree \
+                    port:oidn \
+                    port:tbb \
+                    port:libomp \
+                    port:glew \
+                    port:eigen3 \
+                    port:lzo2 \
+                    port:gflags
```

Comment:
Please sort this (long!) list alphabetically.

Reply:
I can, but I'd like to delay sorting the list alphabetically temporarily until I am able submit an update for one of the newer versions of Blender (see my notes from my OP above). The reason for this is because that list is currently written in the order that is listed in Blender's CMake scripts. Blender has been adding new third-party libraries as dependencies with each release, so I'd like to keep the list in "Blender ordering" until I can get caught up to the latest version.

Reply:
I see.